clang/test/CheckedCRewriter/3d-allocation.c

```diff
@@ -1,4 +1,5 @@
-// RUN: cconv-standalone %s -- | FileCheck -match-full-lines %s
+// RUN: cconv-standalone -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
+// RUN: cconv-standalone %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
```

Comment:
I'm confused what the above is doing. Is it the case that when you write `CHECK` it should apply to both the `-alltypes` and the regular case? Otherwise I'm not sure why this file didn't otherwise change.

Reply:
And: We should be compiling the version of this with `clang` that doesn't use `-alltypes`